java/java.lsp.server/licenseinfo.xml

```diff
@@ -27,6 +27,11 @@
         <file>vscode/src/extension.ts</file>
         <license ref="MIT-vscode-ext" />
     </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/java/lsp/server/refactoring/ui/codicon.ttf</file>
+        <file>src/org/netbeans/modules/java/lsp/server/refactoring/ui/codicon.css</file>
```

Comment:
These files are taken from [Microsoft codicons](https://github.com/microsoft/vscode-codicons) project. Readme mentions it is MIT licensed and points to [standard MIT license](https://opensource.org/licenses/MIT).